packages/admin/admin-rte/src/core/extension/SoftHyphen/EditorComponent.tsx

```diff
@@ -8,16 +9,19 @@ interface IProps {
     children?: React.ReactNode;
 }
 
-const VisibleHyphen = styled("span")`
-    color: #999;
-    &:before {
-        content: "[-]";
-    }
-`;
-
-// children must be rendered, although &shy; is invisible
-// https://github.com/facebook/draft-js/issues/1558
-
 export function EditorComponent(props: IProps): React.ReactElement {
-    return <VisibleHyphen>{props.children}</VisibleHyphen>;
+    return (
+        <Root>
+            <VisibleHyphen> {props.children}</VisibleHyphen>
+        </Root>
+    );
 }
+
+const Root = styled("span")`
+    position: relative;
+`;
+
+const VisibleHyphen = styled(RteSoftHyphen)`
+    position: absolute;
+    color: #999;
+`;
```

Comment:
Why do you need relative and absolute positioning?

Reply:
I don't anymore. It is removed.